dashboard/public/pixelation/pixelation.js

```diff
@@ -37,6 +37,65 @@ function pixelationInit() {
     bitsPerPixelRange.setAttribute("disabled", "true");
     startOver.setAttribute("disabled", "true");
   }
+
+  customizeStyles();
+  initProjects();
+}
+
+function customizeStyles() {
+  if (!window.options) {
+    // Default is version 3 (all features enabled).
+    window.options = {version: '3'};
+  }
+  if (options.version == '1') {
+    $('.hide_on_v1').hide();
+  } else if (options.version == '2') {
```

Comment:
FWIW this is Baker's widget code which I avoided cleaning up during the import.  Since we don't plan to add a lot of new features, I think it's ok to just leave the moved code as-is.